users/views.py

```diff
@@ -210,3 +216,41 @@ def get_context_data(self, **kwargs: Any) -> dict:
         context = super().get_context_data(**kwargs)
         context["is_activated"] = self.activate()
         return context
+
+
+@method_decorator(login_required, name="dispatch")
+@method_decorator(
+    check_permissions(allowed_user_types=[CustomUser.UserType.ADMIN.name, CustomUser.UserType.MANAGER.name]),
+    name="dispatch",
+)
+class NotificationUserListView(ListView):
+    template_name = "user_notifications_list.html"
+    model = CustomUser
+
+    def get_queryset(self) -> QuerySet:
+        return (
+            super()
+            .get_queryset()
+            .filter(
+                Q(is_active=True)
```

Comment:
There is no need to use `Q` if and`&` as in this case this is default `.filter()` behaviour.

Reply:
Ahh, I tested different combinations of querysets and this left after this battlefield :P Deleted.

Reply:
Still there, you don't need to wrap the each filter parts in `Q`.

Reply:
Okaaay, so to sum up we need to wrap filters with `Q` when we use some combinations with `AND` and `OR`?

Reply:
Yes, also `AND` is used by `filter()` by default, so it is used only in some special cases. So `Q` is mostly used when you want to apply `OR`.